pwiz_tools/Skyline/Model/FullScanPropertiesRes.resx

```diff
@@ -277,8 +277,20 @@
   <data name="Description_InjectionTime" xml:space="preserve">
     <value>Injection time used to obtain this spectrum.</value>
   </data>
+  <data name="IsCentroided" xml:space="preserve">
+    <value>Is Centroided</value>
+  </data>
+  <data name="Description_IsCentroided" xml:space="preserve">
+    <value>Is this a centroided or high resolution spectrum.</value>
+  </data>
+  <data name="HighEnergyOffset" xml:space="preserve">
+    <value>High Energy Offset</value>
+  </data>
+  <data name="Description_HighEnergyOffset" xml:space="preserve">
+    <value>High Energy Offset used to obtain the MS2 spectrum.</value>
+  </data>
   <data name="dotp" xml:space="preserve">
-    <value>dotp</value>
+    <value>Library dotp</value>
```

Comment:
I think this should be "Library Dot Product", and the string for "idotp" down below should be "Isotope Dot Product".